config/sara/README.md

```diff
@@ -1,38 +1,53 @@
 # Jurisdiction Configuration
+
 The jurisdictions in `config/sara/jurisdictions.yml` file follow a hierarchical structure. A jurisdiction has a name,
-which is the key and two optional values, `symptoms` and `children`. `symptoms` defines the symptoms
-that the jurisdiction which they belong to would like to track, said jurisdiction will track the
-symptoms that it specifies ***IN ADDITION TO*** the symptoms specified by all of it's parent jurisdictions. The `children` of a jurisdiction are nested jurisdictions that may have their own `symptoms` and/or `children`.
+which is the key and some optional values:
+
+- `phone`, `webpage`, `email` - jurisdiction contact information that will only be shown if populated
```

Comment:
Mention where this information will be shown

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1254/commits/09f765ebf231a8cfc1897e10b648ac8eb5f27be0